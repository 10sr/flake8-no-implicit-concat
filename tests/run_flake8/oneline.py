a = "aaa" "bbb"
b = 'ccc' 'ddd'
c = 'fff' "ggg"
print("abc" "def", "ghi")
