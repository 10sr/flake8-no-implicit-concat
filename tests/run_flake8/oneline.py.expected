./oneline.py:1:10: NIC001 Implicitly concatenated str literals on one line
./oneline.py:2:10: NIC001 Implicitly concatenated str literals on one line
./oneline.py:3:10: NIC001 Implicitly concatenated str literals on one line
./oneline.py:4:12: NIC001 Implicitly concatenated str literals on one line
